observability/monitoring/monitoring-the-gitops-operator-performance.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="monitoring-the-gitops-operator-performance"]
+= Monitoring the {gitops-shortname} Operator performance
+:context: monitoring-the-gitops-operator-performance
+
+toc::[]
+
+The {gitops-title} Operator emits metrics about its performance. With the OpenShift monitoring stack that picks up these metrics, you can monitor and analyze the Operator’s performance. The Operator exposes the following metrics, which you can view by using the {OCP} web console:
+
+.{gitops-shortname} Operator performance metrics
+|===
+|*Metric name* |*Type* |*Description*
```

Comment:
@jaideepr97 I included the type of metrics (gauge/counter), is that beneficial to the user to know?

Reply:
@ekristova I find it beneficial.

@iam-veeramalla This query is pending. Will you be able to help or do we need to wait for @jaideepr97?

Reply:
Users would need to know what they mean for it to be useful
Guage = value can go up or down
Counter = value can only go up

I don't really have strong opinions either way, we can leave them in

Reply:
@ekristova PTAL and do the needful. TY!